.changeset/red-apes-tell.md

```diff
@@ -0,0 +1,24 @@
+---
+'@backstage/create-app': patch
+---
+
+Components `<DefaultResultListItem>`, `<SearchBar>`, `<SearchFilter>`, and `<SearchResult>` are now deprecated in `@backstage/plugin-search` and should be imported from `@backstage/plugin-search-react` instead.
+
+To upgrade your App, update the following in `packages/app/src/components/search/SearchPage.tsx`:
+
+```diff
+import {
+- SearchBar
+- SearchFilter
+- SearchResult
+SearchType,
+- DefaultResultListItem
+} from `@backstage/plugin-search`;
+import {
++ DefaultResultListItem
++ SearchBar
++ SearchFilter
++ SearchResult
+useSearch,
+} from `@backstage/plugin-search-react`;
```

Comment:
as the main change here seems to be where its imported from, what about making the line `from ....` the actual diff?